canonical-websites/www.ubuntu.com
# Added "what is openstack page" and logo grid pattern

## Done

- Added a new "What is OpenStack" page
- Updated the navigation YAML file
- Added new grid pattern style for responsive logos

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/openstack/what-is-openstack](http://0.0.0.0:8001/openstack/what-is-openstack)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Check the page matches the [copydoc](https://docs.google.com/document/d/15xrgIq_KUwHaCyFoI7-yS_JmBbsoyXJ4GpRMFAa7_is/edit) (The diagram will be added next iteration).


## Issue / Card

Fixes #4083
